src/main/java/com/google/cloud/genomics/dataflow/pipelines/ShardedBAMWriting.java

```diff
@@ -90,28 +58,13 @@
  */
```

Comment:
Reminder to us for later, add a link here to the cookbook page with run instructions, like: https://github.com/googlegenomics/dataflow-java/blob/master/src/main/java/com/google/cloud/genomics/dataflow/pipelines/CountReads.java#L59